modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ViewObjectEntriesDisplayContext.java

```diff
@@ -57,7 +60,17 @@ public ViewObjectEntriesDisplayContext(
 		_portletResourcePermission = portletResourcePermission;
 
 		_apiURL = "/o" + restContextPath;
+
 		_objectRequestHelper = new ObjectRequestHelper(httpServletRequest);
+
```

Comment:
I think this code can be moved to the `isObjectWidget` method. Can you do that?

Reply:
Actually, I think we can move this code to the JSP file

Reply:
Done!